extensions/ql-vscode/src/data-extensions-editor/auto-modeler.ts

```diff
@@ -29,18 +29,26 @@ export class AutoModeler {
   ) {}
 
   public async startModeling(
+    dependency: string,
     externalApiUsages: ExternalApiUsage[],
     modeledMethods: Record<string, ModeledMethod>,
     mode: Mode,
   ): Promise<void> {
-    await this.modelDependency(externalApiUsages, modeledMethods, mode);
+    await this.modelDependency(
+      dependency,
+      externalApiUsages,
+      modeledMethods,
+      mode,
+    );
   }
 
   private async modelDependency(
+    dependency: string,
     externalApiUsages: ExternalApiUsage[],
     modeledMethods: Record<string, ModeledMethod>,
     mode: Mode,
   ): Promise<void> {
+    void extLogger.log(`Modeling dependency ${dependency}`);
```

Comment:
Is this logging intentional or maybe leftover from testing?

I'm just wondering if we need it, since most users won't see it in the logs.

Reply:
It was intentional as I thought it might be useful for debugging issues. It's also good to have it right now because otherwise the dependency name isn't used.